docs/guide/usage/mocking-dependency.rst

```diff
@@ -175,6 +175,24 @@ E.g.:
             $this->freeDeliveryThreshold = $threshold;
         }
     }
+    
+    In order to load this custom DI configuration during the test run the test area need to be configured in the Behat test suite so it can load to merge it with the default area.
```

Comment:
I think this line need to start at the beginning (without the indentation) otherwise probably it will be rendered as part of the previous code block